### PR TITLE
Dschz athens

### PR DESCRIPTION
**Pull request by dschz**

### CODE REVIEW

**Comment by zeleGit** [comment]
this is great

**Comment by dschz** [comment]
@zeleGit You are awesome!

File: places/dschz-paris.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Paris, France
+
+### Fun Facts
+
+### Places to Eat
```
Comment (IcyRossa) [inline places/dschz-paris.md]: Where is good to eat?

**Comment by IcyRossa** [comment]
I might go to Greece sometime next year! :smile:

File: places/dschz-paris.md (hunk 1 above)
Comment (dschz) [inline places/dschz-paris.md]: @IcyRossa changes made for restaurants in Paris.

**Pull request merged**